readme.md

```diff
@@ -189,6 +209,34 @@ Image attachment ID or 0.
 
 `wp_calculate_image_srcset()`
 
+##### Usage Example
+
+```
+<?php
+// Remove sources wider than 800px from the 'srcset' for featured images.
+
+function custom_wp_calculate_image_srcset( $sources, $size_array, $image_src, $image_meta, $attachment_id ) {
+	if ( array_key_exists( 'post-thumbnail', $image_meta['sizes'] ) ) {
+		$post_thumbnail_size = array();
```

Comment:
I would probably test if the `$image_src` matches the post-thumbnail file from the `$image_meta` rather than checking for a match of width/height values to make sure you don't accidentally get any false positives for images that happen to match the post-thumbnail size.